pallets/sudo-origin/README.md

```diff
@@ -0,0 +1,70 @@
+# Sudo Module
```

Comment:
that one seems to be outdated - maybe consider removing it for now ?

Reply:
Edited it.